areas/sql/src/AzureMcp.Sql/Services/Models/SqlDatabaseProperties.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Sql.Services.Models
+{
+    /// <summary>
+    /// A class representing the SqlDatabase properties model.
+    /// A database resource properties.
+    /// </summary>
+    internal sealed class SqlDatabaseProperties
```

Comment:
I'm thinking that we may want to remove any properties that we think the user will never query on or be interested in.  Is that possible to determine with some confidence?  (Looking to trim code and save on binary size as a general rule)